config/v1/types_network.go

```diff
@@ -51,6 +51,11 @@ type NetworkSpec struct {
 	// not allowed to be set.
 	// +optional
 	ExternalIP *ExternalIPConfig `json:"externalIP,omitempty"`
+
+	// Service Node port range for services to be exposed on that are of type
```

Comment:
Wording suggestion:

"The port range allowed for Services of type NodePort. If not specified, the default of 30000-32767 will be used. Such Services without a NodePort specified will have one automatically allocated from this range."

Reply:
Can you change this field after installation? ( I believe yes). Probably good to document that.

Reply:
will do